src/stringIndices.js

```diff
@@ -0,0 +1,29 @@
+function getIndex() {
+  let phrase          = document.getElementById('phrase').value
+  let indexNum        = document.getElementById('indexNum').value
+  let currentWord     = 1
+  let nextWord        = phrase.indexOf(' ')
```

Comment:
Consider splitting your phrase into an array of strings using:

```js 
let words = phrase.split( ' ' )
```

With that as a start, I challenge you to refactor to use fewer lines of code, since you can 
`return words[ indexNumber ]`